doc/code.html

```diff
@@ -202,25 +202,37 @@ <h3 id="ImportPaths">Import paths</h3>
 </p>
 
 <p>
-We'll use <code>github.com/user</code> as our base path. Create a directory
-inside your workspace in which to keep source code:
+Create a new directory on your machine in a location you find most convenient, this
+will contain all of will contain all of the source code for our Go program:
 </p>
 
 <pre>
-$ <b>mkdir -p $GOPATH/src/github.com/user</b>
+$ <b>mkdir ~/myproject/hello</b>
 </pre>
 
-
 <h3 id="Command">Your first program</h3>
 
 <p>
 To compile and run a simple program, first choose a package path (we'll use
-<code>github.com/user/hello</code>) and create a corresponding package directory
-inside your workspace:
+<code>github.com/user/hello</code>) and create a new directory on your machine in which your Go program will live. 
+<b>Note:</b> you should ensure this directory lies outwith your <code>$GOPATH</code> unless you have explicitly set
+<code>GO111MODULE</code> to <code>on</code>. You can find more about this here: <a href="https://github.com/golang/go/wiki/Modules#how-to-use-modules">How to Use Modules</a>
+</p>
+w
```

Comment:
don't think this line is intended haha
```suggestion
w
```